### PR TITLE
add Autoloader and ManifestBuilder

### PR DESCRIPTION
**Pull request by darvidor**
Add Autoloader script to load automatically any class or interface in your project.
Add Manifestbuilder class which allows to scan and build manifest.properties.
manifest.properties is a mapping file to match full class name (namespace and class)
with the full path of the file which contains the class.

### CODE REVIEW

File: Manifestbuilder.php
Comment (cosesquemagradenmolt) [inline Manifestbuilder.php:1]: Require to improve the directory exclusion by adding a callbackfunction.

File: build.php
Comment (cosesquemagradenmolt) [inline build.php:1]: require to modify in future release the namespace. Should be sparkbits\adiutor

File: Manifestbuilder.php
Comment (cosesquemagradenmolt) [inline Manifestbuilder.php:1]: equire to modify in future release the namespace. Should be sparkbits\adiutor

**Review by cosesquemagradenmolt — CHANGES_REQUESTED** [review]

File: Manifestbuilder.php
Comment (darvidor) [reply Manifestbuilder.php:1]: we keep the namespace and we'll update in future releases.

**Review by cosesquemagradenmolt — APPROVED** [review]
it is ok.

**Review by cosesquemagradenmolt — APPROVED** [review]

**Pull request merged**